### PR TITLE
Task/APPS-2943 — add notes to case

### PR DESCRIPTION
**Pull request by giuliacampana**
https://user-images.githubusercontent.com/32921059/119686383-22031380-bdfb-11eb-8113-c72b02215ed4.mov

Screen Shot 2021-05-26 at 8 18 42 AM

### CODE REVIEW

File: src/containers/profile/src/sagas/editCRCCase.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    const { entityData } = value;
+    const response :Object = yield call(submitPartialReplaceWorker, submitPartialReplace({ entityData }));
+    if (response.error) throw response.error;
+
+    const crcCaseESID = yield select(selectEntitySetId(CRC_CASE));
+    const propertyFqnsByTypeId = yield select((store :Map) => store.getIn([EDM, PROPERTY_FQNS_BY_TYPE_ID]));
+
+    const crcCaseMap = get(entityData, crcCaseESID);
+    const crcCaseEKID = crcCaseMap ? Object.keys(crcCaseMap)[0] : undefined;
```
Comment (anzioka) [inline src/containers/profile/src/sagas/editCRCCase.js:44]: Could `crcCaseMap` be an empty object? In that case Object.keys(crcCaseMap)[0] could be problematic

Comment (anzioka) [reply src/containers/profile/src/sagas/editCRCCase.js:44]: Update: apparently reading an array index that is out of bounds returns `undefined`

Comment (giuliacampana) [reply src/containers/profile/src/sagas/editCRCCase.js:44]: that's a good point though! i guess no change is needed?

Comment (anzioka) [reply src/containers/profile/src/sagas/editCRCCase.js:44]: Yup!

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**